### PR TITLE
[ZH] Add `Client/Beatmap editor/Design`

### PR DESCRIPTION
**Pull request by Sunnyboy971**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/Design/zh.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+**分别显示不同图层下故事板对象的列表。**点击 “Sprite Library” 来添加故事板元素。可以将新精灵（故事板对象）拖放到对应图层以分配之。新精灵默认位于“前景”图层。点击精灵名会显示并跳转到精灵的位置及时间帧。复制的精灵使用的名称与原精灵相同。使用 `Delete` 键或顶部菜单的编辑 -> 删除来删除精灵。
+
+**注意** “>” 表示图层遮挡顺序：物件 > 前景 > 通过时/失败时 > 背景。
+
+#### 精灵库 (Sprite Library)
+
+![所使用精灵的视觉表现。](img/SE_SL.jpg "所使用精灵的视觉表现。")
+
+点击 "Browse..." 按钮来搜索要使用的精灵。你可以在精灵库窗口中点击缩略图来复制精灵。
```
Comment (YumeMuzi) [inline wiki/Client/Beatmap_editor/Design/zh.md:142]:
```suggestion
#### 精灵图库 (Sprite Library)

![所使用精灵图的视觉表现。](img/SE_SL.jpg "所使用精灵图的视觉表现。")

点击 "Browse..." 按钮来搜索要使用的精灵图。你可以在精灵图库窗口中点击缩略图来复制精灵图。
```

File: wiki/Client/Beatmap_editor/Design/zh.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### 中央 （谱面故事板）
+
+这是**故事板和基于时间线与图层开关的改变的视觉表现。** 在这里放置故事板对象、制作故事板。
+
+### 中央右侧 （对象）
+
+![谱面中不同条件的图层使用的故事板元素列表](img/SE_OBJ.jpg "谱面中不同条件的图层使用的故事板元素列表")
+
+**分别显示不同图层下故事板对象的列表。**点击 “Sprite Library” 来添加故事板元素。可以将新精灵（故事板对象）拖放到对应图层以分配之。新精灵默认位于“前景”图层。点击精灵名会显示并跳转到精灵的位置及时间帧。复制的精灵使用的名称与原精灵相同。使用 `Delete` 键或顶部菜单的编辑 -> 删除来删除精灵。
```
Comment (YumeMuzi) [inline wiki/Client/Beatmap_editor/Design/zh.md:134]:
```suggestion
**分别显示不同图层下故事板对象的列表。**点击 “Sprite Library” 来添加故事板元素。可以将新精灵图（故事板对象）拖放到对应图层以分配之。新精灵图默认位于“前景”图层。点击精灵名会显示并跳转到精灵图的位置及时间帧。复制的精灵图使用的名称与原精灵图相同。使用 `Delete` 键或顶部菜单的编辑 -> 删除来删除精灵图。
```

File: wiki/Client/Beatmap_editor/Design/zh.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+##### 变换效果 （基于时间帧）
+
+| 命令 | 用法 |
+| :-- | :-- |
+| 移动 | 翻译\[到哪里去（比如上移）\]。 |
+| 缩放 | 成比例缩放（正方形仍然是正方形）。 |
+| 淡入淡出 | 淡入/淡出。（变暗对变亮） |
+| 旋转 | 旋转一定的“弧度”（不是角度）。 |
+| 色块模式 | 强制为精灵上色。可以通过在终点使用不同的颜色来逐步改变元素的颜色。“永久效果”，这是基本解释。 |
```
Comment (YumeMuzi) [inline wiki/Client/Beatmap_editor/Design/zh.md:100]:
```suggestion
| 色块模式 | 强制为精灵图上色。可以通过在终点使用与起点不同的颜色，来逐步改变元素的颜色，“永久效果”。这是基本解释。 |
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
精灵 -> 精灵图

**Review by TicClick — APPROVED** [review]

**Pull request merged**